### ISSUE TITLE
'Internal type checking error' when file contains UTF-8 non-breaking space

### ISSUE THREAD

**Issue by MikhailArkhipov** [microsoft/pyright#409]
**Describe the bug**
'Internal type checking error' when file contains UTF-8 non-breaking space

**To Reproduce**
- Create an empty Python file
- Paste
```python
import sys
```
where between `import` and `sys` space is actually `0xC2A0` (non-breaking UTF-8 space). This happens when copying code from a Web site.
- Observe squiggly and internal error message.

**Expected behavior**
No internal errors and indication that non-ASCII space character is not valid.

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/12820357/70590231-ec2a8a80-1b86-11ea-868a-a32b0de4b2c3.png)

**Comment by erictraut** [comment]
Thanks for the bug report. Here's a fix: https://github.com/microsoft/pyright/commit/bd4746940d69752256779befd4e5a27bdc73add9

**Comment by erictraut** [comment]
This is now fixed in version 1.1.10, which I just published.
